src/index.html

```diff
@@ -16,17 +16,38 @@
       href="style.css"
     />
```

Comment:
HTML Formatting: The attributes of the `link` tag should be correctly formatted. Each attribute should start on a new line with 2-space indentation related to the tag.